### ISSUE TITLE
ionic how to call first handleOpenURL & after call state provider otherwise url

### ISSUE THREAD

**Issue by vishalwayachal** [EddyVerbruggen/Custom-URL-scheme#219]
i am using this package in ionic but when i click on link (app running first time) first redirect to state provider otherwise url after that call handleOpenURL 
is any solution how to call first handleOpenURL & after call state provider otherwise url

**Comment by sliard** [comment]
Good point, same pb for me. Not with ionic but just angularJS, handleOpenURL is call after stateProvider router, the fist time I run app :/

Any solution to map scheme with "page url" use by stateProvider ?
